syntax = "proto3";

package templateProto;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply);
    rpc TimeWorkServer (Empty) returns (Seconds);
}

message Empty {
}

message Seconds {
    int32 time = 1;
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string answer = 1;
}